### PR TITLE
debugger toolbar

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/editor/web/js/debugger/frames.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    var instance;
+
+    var Frames = function (){
+
+    };
+
+    Frames.prototype = Object.create(EventChannel.prototype);
+    Frames.prototype.constructor = Frames;
+
+    Frames.prototype.render = function (container) {
```
Comment (sumuditha-viraj) [inline modules/editor/web/js/debugger/frames.js]: Let's add Doc comments

File: modules/editor/web/css/debugger.css [hunk 2]
```diff
@@ -15,7 +14,7 @@
 
 /*TODO remove this when feature is done*/
 .left-tool-bar .debugger-activate-btn, .left-tool-bar .debugger-activate-btn i{
-    display: none;
+    /*display: none;*/
```
Comment (pahans) [inline modules/editor/web/css/debugger.css]: removed

File: modules/editor/web/css/debugger.css [hunk 3]
```diff
@@ -2,11 +2,10 @@
     color: #dedede;
 }
 
-.debugger-container .panel-heading{
+.debugger-container .panel-heading {
     padding: 2px 1px;
-    background: whitesmoke;
+    background: #333333;
```
Comment (pahans) [inline modules/editor/web/css/debugger.css]: fixed in https://github.com/ballerinalang/composer/pull/2007

File: modules/editor/web/js/debugger/frames.js (hunk 1 above)
Comment (pahans) [inline modules/editor/web/js/debugger/frames.js]: fixed with https://github.com/ballerinalang/composer/pull/1996

**Pull request merged**